### DISCUSSION TITLE
Projects (beta) ) - possibility to distribute issues across views by moving and directly from issues

### DISCUSSION THREAD

**Discussion by katrinadowl** [community/community#8462] [Projects and Issues]
Can you make possible to distribute issues by views as it was in old Projects with cards?

### Problem description

I would like to distribute issues by type of work (one type of work - one view). But it is not possible for me now in Beta Projects because of reasons:

1. Now it is possible to distribute issues by views only using label or repository name. 
   I cannot use this way because It is not possible for me to create label I need in company repositories. We have some rules regarding labeling depending on repo.

2. It is not convenient to distribute by views by repos. I have tasks related to one category in different company repos. 

### Solution:
Add possibility to distribute issues by view directly from issues and by moving issues between views as it was in old Projects using cards.

**Comment by Gotterbild** [comment]
Yes, it will be super handy if Project Views to be a separate independent tabs with their own issues, without the need to use filters by labels, repo, etc.

I've met many cases of using views as separate tabs and people always have to use filter as some kind of a hack. It seems to me it should be the default behavior to have views as separate tabs.

Examples I've seen:
![image](https://user-images.githubusercontent.com/23502743/144855446-6f777ffc-da25-405d-acf6-661cb25b4d76.png)
![image](https://user-images.githubusercontent.com/23502743/144855531-ff8d6594-87de-4ae3-9265-877287b919a6.png)
![image](https://user-images.githubusercontent.com/23502743/144855638-72a9cb96-1373-4386-b73d-75533c3a98d2.png)
